src/oncall/ui/__init__.py

```diff
@@ -73,12 +73,12 @@
 HEADER_COLOR = None
 IRIS_PLAN_SETTINGS = None
 USERCONTACT_UI_READONLY = None
-LOGIN_SPLASH = None
+LOGIN_REQUIRED = None
 
 
 def index(req, resp):
     user = req.env.get('beaker.session', {}).get('user')
-    if str(user) == 'None' and LOGIN_SPLASH:
+    if str(user) == 'None' and LOGIN_REQUIRED:
```

Comment:
Why not do `if user is None` instead of casting to a string? Or are there cases where the value of user will literally be the string `'None'`?

Reply:
I have no idea. Probably we should change it to `is None`. @diegocepedaw, any reason for `'None'`?